woocommerce/woocommerce-gateway-stripe
# Add additional tests for payment capture failure cases

## Changes proposed in this Pull Request:

This PR is a follow-up to [this comment](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2211#pullrequestreview-820354571). PR #2211 added an API endpoint for capturing payment intents — there are a number of ways that this can fail, so this PR adds test coverage for the failure cases that weren't covered by the original PR.

## Testing instructions

- Verify that tests pass (`npm run test:php`).
- To test these two error cases manually (refunded orders and capture failure), see Test Case 3 and Test Case 5 in #2211.

---


**Post merge**
